libvast/vast/schema.hpp

```diff
@@ -77,6 +78,7 @@ class schema : detail::equality_comparable<schema> {
 
   friend void serialize(caf::serializer& sink, const schema& sch);
   friend void serialize(caf::deserializer& source, schema& sch);
+  friend void serialize(detail::legacy_deserializer& source, schema& sch);
```

Comment:
Rename to `inspect` so this works with the legacy deserialiser. `serialize` is very much outdated, and predated inspect.

Reply:
Resolved in c0311cfd71c165a71bc0e8c4f941fa938572e14d